packages/zapp/console/src/components/Executions/ExecutionDetails/ExecutionNodeViews.tsx

```diff
@@ -146,19 +144,20 @@ export const ExecutionNodeViews: React.FC<ExecutionNodeViewsProps> = ({ executio
     );
   };
 
-  const renderTab = (tabType) => (
-    <WaitForQuery
-      errorComponent={DataError}
-      query={childGroupsQuery}
-      loadingComponent={LoadingComponent}
-    >
-      {() => <ExecutionTab tabType={tabType} abortMetadata={abortMetadata ?? undefined} />}
-    </WaitForQuery>
-  );
-
-  if (loading) {
-    return <LoadingComponent />;
-  }
+  const renderTab = (tabType) => {
+    if (loading) {
```

Comment:
nit: this is hard to read, can we rename loading to nodeExecutionsLoading or something?